recipes/prismatic/meta.yaml

```diff
@@ -0,0 +1,164 @@
+        - {{ compiler('cxx') }}              
+        - {{ cdt('libxext-devel') }}         # [linux]
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('libxau-devel') }}          # [linux]
+        - {{ cdt('libselinux-devel') }}      # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - make                               # [unix]
+        - cmake
+      host:
+        - boost
+        - hdf5
+        - fftw
+        - zlib
+      run:
+        - zlib
+        - hdf5
+        - fftw
+
+
+    test:
+      source_files:
+        - SI100.XYZ
+      commands:
+        - prismatic -i SI100.XYZ
+
+
+  - name: prismatic_gui
+    build:
+      skip: True  # [win]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+      script: bash ${RECIPE_DIR}/combined_build_install.sh gui # [not win]
+
+    requirements:
+      build:
+        - {{ compiler('c') }}                
+        - {{ compiler('cxx') }}              
+        - {{ cdt('libxext-devel') }}         # [linux]
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('libxau-devel') }}          # [linux]
+        - {{ cdt('libselinux-devel') }}      # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - make                               # [unix]
+        - cmake
+      host:
+        - boost
+        - hdf5
+        - fftw
+        - qt 5.9.*
+        - h5py
+        - zlib
+      run:
+        - zlib
+      
+    test:
+      commands:
+        - command -v prismatic-gui  # [linux]
+        - test -f $PREFIX/Applications/prismatic-gui.app/Contents/MacOS/prismatic-gui  # [osx]
+
+  - name: pyprismatic
+    build:
+      skip: True  # [win or py2k or osx]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+      script: bash ${RECIPE_DIR}/combined_build_install.sh py # [not win]
+
+    requirements:
+      build:
+        - python
+        - pip
+        - {{ compiler('c') }}                
+        - {{ compiler('cxx') }}              
+        - {{ cdt('libxext-devel') }}         # [linux]
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('libxau-devel') }}          # [linux]
+        - {{ cdt('libselinux-devel') }}      # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - make                               # [unix]
+        - cmake
+      host:
+        - python
+        - boost
+        - hdf5
+        - fftw
+        - qt 5.9.*
```

Comment:
Do you need to use QT 5.9 exactly? Only 5.12 is being built at the moment and using old versions can become painful quite quickly (due to dependencies being incompatible and not built).

Reply:
We need qt 5.9.* for linux (not OSX), due to a missing dependency in 5.12 ([see this commit](https://github.com/conda-forge/staged-recipes/pull/9834/commits/8bbdd7618620167faafc5143b75f6b82173db634)). I see that there's been a new release of 5.12 since then - which should solve the libEGL.so problem. I'll remove the version specification, and see if it works now.

Reply:
The build does now work with 5.12 - so we will use that for both OSX & linux.